project/src/components/offers-sorting/offers-sorting.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+import { useAppDispatch, useAppSelector } from '../../hooks/useAppDispatch';
+import { setSortTypeAction } from '../../store/actions';
+import { SortType } from '../../types/types';
+
+type OffersSoringType = {
+  options: SortType[];
+}
+
+export const OffersSorting = ({options}: OffersSoringType): JSX.Element => {
+  const [isOpened, setOpened] = useState(false);
```

Comment:
по договоренности сеттер всегда содержит имя переменной стейта

```suggestion
  const [isOpened, setIsOpened] = useState(false);
```